BUILD.gn

```diff
@@ -123,7 +123,7 @@ if (current_toolchain != "${dir_pw_toolchain}/default:default") {
         if (enable_pylib) {
           deps += [ "${chip_root}/src/pybindings/pycontroller" ]
         }
-        deps += [ "${chip_root}/src/controller/python" ]
+        deps += [ "${chip_root}/src/controller/python:chip-repl" ]
```

Comment:
This won't automatically build `chip-library` too though would it?

Reply:
Right, I had a dependency in the chip-repl target a earlier version, but had to remove it for reasons I don't remember. I'll need to check that again.